zebrad/src/components/mempool/storage.rs

```diff
@@ -21,6 +24,8 @@ pub enum State {
     /// An otherwise valid mempool transaction was mined into a block, therefore
     /// no longer belongs in the mempool.
     Confirmed(block::Hash),
+    /// Rejected because it conflicted with another transaction already in the mempool.
+    Conflict,
```

Comment:
```suggestion
    /// Rejected because its spends conflicted with another transaction already in the mempool.
    SpendConflict,
```

Reply:
Oh right, I had fixed that on my branch and also tweaked some other documentation and renamed the `has_conflicts` method to `has_spend_conflicts` :sweat_smile:. In theory it's generic enough to detect any kind of conflict, given the appropriate extractor, but I think it's best to keep it specific for now and change it if only if we find another use case for it :thinking: What do you think?

Reply:
I think it's best to name things after what they do right now.
(If we make big changes to the behaviour later, we can change the name then.)